contents/blog/posthog-alternatives.md

```diff
@@ -0,0 +1,727 @@
+---
+title: "In-depth: PostHog alternatives, compared"
```

Comment:
Interested on headline ideas here. Some other ideas:

- PostHog Alternatives: What users say
- What users say about popular PostHog alternatives
- The most popular PostHog alternatives, compared

Reply:
Tried to think of what would be best, but didn't get anything I felt strongly about. As long as PostHog alternatives is in the title, should be fine. Current title is a little awkward because of the double pauses (one for colon, one for comma), so I would change that if anything.
- Popular PostHog alternatives: What users say
- Comparing PostHog alternatives (with user reviews)
- User reviews of PostHog alternatives